### PR TITLE
add chart fields to admin 3 and update storm layers; COUNTRY=myanmar

### PR DESCRIPTION
**Pull request by wadhwamatic**
### Description

This fixes missing admin codes for charts in adm3 boundary file, and reconfigure ADAM layers

## How to test the feature:

Charts:
- load 10-day rainfall layer
- click on an admin area
- select a chart from the tooltip

ADAM layers:
- load a tropical storm layer
- run exposure analysis on the results

## Checklist - did you ...

Test your changes with

- [x] `REACT_APP_COUNTRY=rbd yarn start`
- [x] `REACT_APP_COUNTRY=cambodia yarn start`
- [x] `REACT_APP_COUNTRY=mozambique yarn start`
- [x] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by wadhwamatic** [comment]
PR 1000!

File: frontend/src/config/myanmar/layers.json [hunk 1]
```diff
@@ -2399,7 +2399,7 @@
     "opacity": 0.7,
     "legend": [],
     "legend_text": "Tropical Storms - Nodes",
-    "base_url": "https://sparc.wfp.org/geoserver",
```
[the remark this answers is not in the document]
Comment (wadhwamatic) [reply frontend/src/config/myanmar/layers.json:2402]: Yes, but I'm working on these one by one as some issues are coming up on the geoserver side

**Review by ericboucher — APPROVED** [review]

**Comment by ericboucher** [comment]
@wadhwamatic the code only allows charts for admin 0-1-2 atm, should we add admin 3?

**Comment by wadhwamatic** [comment]
> @wadhwamatic the code only allows charts for admin 0-1-2 atm, should we add admin 3?

@ericboucher - we still only provide up to admin2 level data in the HDC API. I added the IDs to the admin 3 boundary file because the chart data fetch from the tooltip was failing without it.

**Pull request merged**